### DISCUSSION TITLE
Examples and MapLibre

### DISCUSSION THREAD

**Discussion by roman-hatz-srf** [uber/react-map-gl#1967] [General]
Hi 

We are evaluating this library for production use. Looks very promising. 
 
We produce and host all map data (tiles) in house and don't need to buy it. 
We want to use react-map-gl with maplibre-gl. As documented we include mapbox-gl with the dummy dependency. 
"mapbox-gl": "npm:empty-npm-package@1.0.0"

Are there any problems we should be aware of when using react-map-gl without Mapbox? 

Thanks.

**Comment by Pessimistress** [comment]
This library currently uses the TypeScript definitions of Mapbox for both Mapbox and Maplibre. Maplibre 2.0 API has diverged from Mapbox, so some newer methods may not have the same signature. Other than that, everything should function exactly the same.
